opentelemetry-semantic-conventions/src/attribute.rs

```diff
@@ -1,60 +1,79 @@
 // DO NOT EDIT, this is an auto-generated file
 //
 // If you want to update the file:
-// - Edit the template at scripts/templates/semantic_attributes.rs.j2
+// - Edit the template at scripts/templates/registry/rust/attributes.rs.j2
```

Comment:
The attributes.rs file currently defines semconv constants, which are then re-exported in various other modules. If there aren't any (or too many) shared definitions used across multiple modules (i.e trace.rs, metrics.rs, resource.rs), would it make sense to move these constants to their respective modules where they actually belong? The attributes.rs can grow over time as we add more definitions, and add the semconv for logs later, could be difficult to manage even though auto-generated.

I know this has been the existing behavior, but then right time to decide any such change.

Reply:
Just to add - this is not a blocker for the PR, something to discuss and plan for subsequent if all agree.